elastic/kibana
# [Security Solutions] Update risk score tables to filter by timerange

issue: https://github.com/elastic/kibana/issues/162451

## Summary

* Update Entity analytics dashboard to filter by timerange and to display timestamp field
* Update Users risk score tab to filter by timerange and to display timestamp field
* Update Hosts risk score tab to filter by timerange and to display timestamp field
* Delete tooltip that used to warn users that risk tables din't filter by timerange


<img width="1501" alt="Screenshot 2023-10-13 at 11 54 19" src="https://github.com/elastic/kibana/assets/1490444/a99e6ec7-0cbd-44a9-b1b1-b2dc9f4ad7cf">
<img width="1506" alt="Screenshot 2023-10-13 at 11 54 38" src="https://github.com/elastic/kibana/assets/1490444/78f59c54-9210-4d09-8e22-bdab1b2103c5">
<img width="1497" alt="Screenshot 2023-10-13 at 11 54 53" src="https://github.com/elastic/kibana/assets/1490444/35c19ee4-3cbc-42f5-96c1-1c63dc47300b">



### How to test
* Create alerts and enable the risk engine
* Check if the Entity analytics dashboard filters by timerange
* Check if the Users risk score tab filters by timerange
* Check if the Hosts risk score tab filters by timerange
* Check if the risk score on top of the user details page does NOT filter by timerange
* Check if the risk score inside flyouts (Alerts, users and hosts) does NOT filter by timerange
* Check if the info tooltips about the timerange filter were removed



### Checklist
